envoy/network/connection.h

```diff
@@ -30,6 +30,7 @@ enum class ConnectionEvent {
   RemoteClose,
   LocalClose,
   Connected,
+  ConnectedZeroRtt,
```

Comment:
This seems a bit independent of the PersistentQuicInfo refactor. Should it go into a different CL?

Reply:
I'm not so sure about splitting it out because it allows me to test the refactored session cache lifetime actually works for 0-RTT.